pkg/rehearse/jobs_test.go

```diff
@@ -330,7 +320,17 @@ func TestExecuteJobsErrors(t *testing.T) {
 				t.Fatalf("Failed to setup watch: %v", err)
 			}
 			fakecs.Fake.PrependWatchReactor("prowjobs", makeSuccessfulFinishReactor(watcher, tc.jobs))
-			fakecs.PrependReactor("create", "prowjobs", tc.reactor)
+			fakecs.Fake.PrependReactor("create", "prowjobs", func(action clientgo_testing.Action) (bool, runtime.Object, error) {
+				createAction := action.(clientgo_testing.CreateAction)
+				pj := createAction.GetObject().(*pjapi.ProwJob)
+				for _, jobName := range tc.failToCreate {
```

Comment:
If you use `sets.String` this becomes `tc.failToCreate.Has(pj.Spec.Job)`

Reply:
Sweet. It'll take a while until I memorize all the kube apis.